resources/eventing/charts/controller/values.yaml

```diff
@@ -81,3 +81,13 @@ healthProbe:
     periodSeconds: 3
     successThreshold: 1
     timeoutSeconds: 2
+
+preUpgradeJob:
+  releaseName: "1.24.3"
+  kymaNamespace: "kyma-system"
+  eventingControllerName: "eventing-controller"
```

Comment:
Why do we have these two names here? Shouldn't we get them at runtime?

Reply:
I have updated it